### DISCUSSION TITLE
mid-crossfade update to animation-crossfade-duration property

### DISCUSSION THREAD

**Discussion by gravytates** [google/model-viewer#2843] [Q&A]
Hello, it looks like all animation crossfades progress linearly from A to B. I'm trying to change this linear progression to one with a cubic bezier curve. Without a niche little method, I was attempting to update the `animation-crossfade-duration` property during the period of a crossfade. 
Example:
initial duration: 1000ms, initial period: 250ms (25% of total crossfade complete)
second duration: 500ms, second period: 250ms (75% of total crossfade complete)
third duration: 1000ms, third period: 250ms (100% of total crossfade complete)
total duration: 750ms

However, it seems that updating the property mid-crossfade does not have an effect on the current crossfade. Is there another way to achieve this goal?
Thanks in advance!

**Comment by ghost** [comment]
Can you provide a glitch example of what you are trying?
https://glitch.com/edit/#!/model-viewer

**Reply by gravytates** [reply]
sure thing 
https://glitch.com/edit/#!/spicy-endurable-purchase
